free-1.md

```diff
@@ -57,12 +153,26 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Probably upwards of 70% of their time on the systems aspect.  While many of 
+the reaction mechanisms are straightforward, data structure design would be
+very non-trivial, as would be transcribing the mechanisms to algorithms.  The 
+language would probably be relatively trivial.
```

Comment:
You are correct - it is absolutely not trivial.  I just think that the difficulty of actually implementing all of the different reactions (i.e. the systems side) is so much more difficult that the language design seems trivial beside it.